styles/globals.css

```diff
@@ -16,6 +16,13 @@ a {
 * {
   box-sizing: border-box;
 }
+.sortIcon {
```

Comment:
Please keep styling relevant to the component

Reply:
ok.